### PR TITLE
Create job feature

### PR DESCRIPTION
**Pull request by johnamielyasis**
omg it keeps changing 

- adds some global styles to get rid of empty white space
- adds job post feature page 
- fixes some bugs caused by auto merge

### CODE REVIEW

File: client/components/CreateJob.jsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+`;
+
+const TextArea = styled.textarea`
+  margin: 10px 0;
+  height: 120px;
+  width: 100%;
+`;
+
+const initialForm = {
```
Comment (claire123c) [inline client/components/CreateJob.jsx:69]: Is this initial form needed? Looks like the user can't submit the form if the required inputs aren't filled out anyway.

File: client/components/CreateJob.jsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { theme } from '../src/constants';
```
Comment (claire123c) [inline client/components/CreateJob.jsx:3]: Nice abstracting out for constant themes!

File: client/components/CreateJob.jsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  > * {
+    box-sizing: border-box;
+  }
+`;
+
+const Select = styled.select`
+  margin: 20px 0;
+  height: 30px;
+  width: 100%;
+
```
Comment (claire123c) [inline client/components/CreateJob.jsx]: extra whitespace

File: client/components/CreateJob.jsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+const Submit = styled.button`
+  margin: 10px 0;
+  height: 30px;
+  width: 150px;
+  background-color: ${theme.colors.Independence};
+  color: white;
+  font-weight: normal;
+  border-radius: 2px;
+
```
Comment (claire123c) [inline client/components/CreateJob.jsx]: whitespace

File: client/components/PostAJob.jsx [hunk 5]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import styled from 'styled-components';
+import CreateJob from './CreateJob.jsx';
 
 const Background = styled.div`
   background-color: #274358;
```
Comment (claire123c) [inline client/components/PostAJob.jsx:6]: do you want to use your theme file for this?

File: generateData/sampleData/generateData.js [hunk 6]
```diff
@@ -8,6 +8,7 @@
 */
 
 const data = require('./json.js');
+console.log(data)
```
Comment (claire123c) [inline generateData/sampleData/generateData.js]: don't forget to remove console.logs, so others won't have to!

File: client/src/constants/theme.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+  Xiketic: '#14080E',
+  Independence: '#49475B',
+  SteelTeal: '#799496',
+  LaurelGreen: '#ACC196',
+  GreenYellow: '#E9EB9E'
+}
+
+export const theme = {
+  colors: {
+    ...pal,
```
Comment (claire123c) [inline client/src/constants/theme.js:11]: nice use of spread operator

File: client/components/CreateJob.jsx [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+`;
+
+const Select = styled.select`
+  margin: 20px 0;
+  height: 30px;
+  width: 100%;
+
+`;
+
+const Submit = styled.button`
```
Comment (claire123c) [inline client/components/CreateJob.jsx:53]: If you want to make the button consistent with the login page - https://material-ui.com/components/buttons/

**Review by claire123c — APPROVED** [review]
Looks great!

File: client/components/CreateJob.jsx (hunk 1 above)
Comment (johnamielyasis) [reply client/components/CreateJob.jsx:69]: it's what resets the form after a successful submit

File: generateData/sampleData/generateData.js (hunk 6 above)
Comment (johnamielyasis) [reply generateData/sampleData/generateData.js]: will do, thank ou

**Pull request merged**